src/test/scala/mesosphere/marathon/api/v2/json/V2GroupUpdateTest.scala

```diff
@@ -132,4 +132,25 @@ class V2GroupUpdateTest extends FunSuite with Matchers with GivenWhenThen {
       update(V2Group(Group.empty), Timestamp.now())
     }
   }
-}
+
+  test("Relative path of a dependency, should be relative to group and not to the app") {
+    Given("A group with two apps. Second app is dependend of first.")
+
+    val update = V2GroupUpdate(PathId.empty, Set.empty[V2AppDefinition], Set(
+      V2GroupUpdate(
+        "test-group".toPath,
+        Set(V2AppDefinition("test-app1".toPath),
+          V2AppDefinition("test-app2".toPath, dependencies = Set("test-app1".toPath)))
+      )
+    ))
```

Comment:
I hope you meant the line feed before 'When'. I matched it up with the style of the other tests.